sdk/batch/Microsoft.Azure.Batch.Conventions.Files/src/JobOutputStorage.cs

```diff
@@ -49,64 +47,30 @@ public class JobOutputStorage
         /// <remarks>The container must already exist; the JobOutputStorage class does not create
         /// it for you.</remarks>
         public JobOutputStorage(Uri jobOutputContainerUri)
-            : this(CloudBlobContainerUtils.GetContainerReference(jobOutputContainerUri), null)
+            : this(CloudBlobContainerUtils.GetContainerReference(jobOutputContainerUri))
         {
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="JobOutputStorage"/> class from a storage account
         /// and job id.
         /// </summary>
-        /// <param name="storageAccount">The storage account linked to the Azure Batch account.</param>
+        /// <param name="blobClient">The blob service client linked to the Azure Batch Storage Account.</param>
         /// <param name="jobId">The id of the Azure Batch job.</param>
         /// <remarks>The job output container must already exist; the JobOutputStorage class does not create
         /// it for you.</remarks>
-        public JobOutputStorage(CloudStorageAccount storageAccount, string jobId)
-            : this(CloudBlobContainerUtils.GetContainerReference(storageAccount, jobId), null)
+        public JobOutputStorage(BlobServiceClient blobClient, string jobId)
+            : this(CloudBlobContainerUtils.GetContainerReference(blobClient, jobId))
         {
         }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="JobOutputStorage"/> class from a URL representing
-        /// the job output container.
-        /// </summary>
-        /// <param name="jobOutputContainerUri">The URL in Azure storage of the blob container to
-        /// use for job outputs. This URL must contain a SAS (Shared Access Signature) granting
-        /// access to the container, or the container must be public.</param>
-        /// <param name="storageRetryPolicy">The retry policy for storage requests.</param>
-        /// <remarks>The container must already exist; the JobOutputStorage class does not create
-        /// it for you.</remarks>
-        public JobOutputStorage(Uri jobOutputContainerUri, IRetryPolicy storageRetryPolicy)
```

Comment:
Current Storage SDK does not support interoperability with Retry Policies. You can specify retry options on the BlobServiceClient directly. This will be mentioned in the Changelog.